opensearch/chart/alerts/opensearch-cluster.yaml

```diff
@@ -0,0 +1,88 @@
+# All alert severities are set to 'info' for setup only. Original severity is preserved as a comment on each line.
+groups:
+  - name: opensearch-cluster
+    interval: 30s
+    rules:
+      - alert: OpenSearchClusterRed
+        expr: opensearch_cluster_status == 2
+        for: 5m
+        labels:
```

Comment:
We need the possibility here to add our custom labels. [See some example here.](https://github.com/cloudoperators/greenhouse-extensions/blob/main/thanos/charts/alerts/compactor.yaml#L13)

Reply:
I use `{{- with` and `toYaml` in line 13. But `tpl` is also nice in case we want templating in the `additionalRuleLabels`. Changed it.

Reply:
Thanks :). Pls fix the linting.